webapp/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <v-container class="fill-height">
+        <v-responsive class="d-flex align-center text-center fill-height">
+            <v-img height="300" src="@/assets/logo.svg" />
+
+            <div class="text-body-2 font-weight-light mb-n1">Welcome to</div>
+
+            <h1 class="text-h2 font-weight-bold">Vuetify</h1>
+
+            <div class="py-14" />
+
+            <v-row class="d-flex align-center justify-center">
+                <v-col cols="auto">
+                    <v-btn
+                        href="https://vuetifyjs.com/components/all/"
+                        min-width="164"
+                        rel="noopener noreferrer"
+                        target="_blank"
+                        variant="text"
+                    >
+                        <v-icon icon="mdi-view-dashboard" size="large" start />
+
+                        Components
+                    </v-btn>
+                </v-col>
+
+                <v-col cols="auto">
+                    <v-btn
+                        color="primary"
+                        href="https://vuetifyjs.com/introduction/why-vuetify/#feature-guides"
+                        min-width="228"
+                        rel="noopener noreferrer"
+                        size="x-large"
+                        target="_blank"
+                        variant="flat"
+                    >
+                        <v-icon icon="mdi-speedometer" size="large" start />
+
+                        {{ t("Track Types") }}
```

Comment:
I think we should use keys instead of `en_US` source strings. Or at least use some kind of structure to sort the strings.

For example:
```js
t('the-path.to-the-file.TheComponentName.the-translation-key')
```

Reply:
Agreed. We would need to change how the files appear in Weblate.

Reply:
This app will appear as a different component in Weblate. Don't worry about matching legacy - the strings will all need re-translating anyway

Reply:
Okay. I've removed all the HTML from the keys in the translation files.